dotnet/src/Connectors/Connectors.HuggingFace/Core/Models/ImageToTextGenerationResponse.cs

```diff
@@ -2,11 +2,11 @@
 
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
-using static Microsoft.SemanticKernel.Connectors.HuggingFace.Client.TextGenerationResponse;
+using static Microsoft.SemanticKernel.Connectors.HuggingFace.Client.Models.TextGenerationResponse;
 
 #pragma warning disable CA1812 // Avoid uninstantiated internal classes
 
-namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Client;
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Client.Models;
```

Comment:
not ` Microsoft.SemanticKernel.Connectors.HuggingFace.Core` ?
I thought we use only three namespaces in connectors:
1.  `Microsoft.SemanticKernel`
2. ` Microsoft.SemanticKernel.Connectors.<connectorName>`
3.  `Microsoft.SemanticKernel.Connectors.<connectorName>.Core`

Reply:
Fixed namespaces.